src/App.js

```diff
@@ -31,6 +42,10 @@ function App() {
             newFavorites = favouriteItems.concat(product);
         }
         setFavouriteItems(newFavorites);
+        window.localStorage.setItem(
+            'favouriteProduct',
+            JSON.stringify(newFavorites)
+        );
```

Comment:
You could write another `useEffect` that runs, when `favoriteItems` is changed.
Here is a [good article](https://blog.logrocket.com/using-localstorage-react-hooks/) about that topic.

Reply:
thanx